### DISCUSSION TITLE
User IP address

### DISCUSSION THREAD

**Discussion by thomaskamalakis** [Budibase/budibase#9730] [Help]
Hi!

  Wild question: is there any way to have access to request data on Budibase? I need to store the user's IP address in the database when an object is created.

  Thanks!

  Thomas

**Comment by melohagan** [comment]
Hey @thomaskamalakis 

In the coming weeks the new release will include a paid feature called Audit Logs. This will allow you to store a log of user actions. 

IP addresses will not be stored by default, but this can be set through an environment variable for self-hosted users.
